src/kibana/components/vislib/lib/y_axis.js

```diff
@@ -15,8 +15,8 @@ define(function (require) {
      */
     function YAxis(args) {
       this.el = args.el;
-      this.yMin = args.yMin;
-      this.yMax = args.yMax;
+      this.yMin = args.yMin || 0;
+      this.yMax = args.yMax || 0;
```

Comment:
@stormpython I'm not sure if the default yMin and yMax values are correct - I can adjust if not (maybe they should be -1 and 1?)